rest_framework/schemas.py

```diff
@@ -467,31 +466,36 @@ def get_description(self, path, method, view):
         This will be based on the method docstring if one exists,
         or else the class docstring.
         """
-        method_name = getattr(view, 'action', method.lower())
-        method_docstring = getattr(view, method_name, None).__doc__
-        if method_docstring:
+        method = method.lower()
+        method_name = getattr(view, 'action', method)
+        view_method = getattr(view, method_name, None)
+        description = None
+        if view_method:
             # An explicit docstring on the method or action.
-            return formatting.dedent(smart_text(method_docstring))
-
-        description = view.get_view_description()
+            description = api_settings.VIEW_DESCRIPTION_FUNCTION(view_method)
+        if not description:
```

Comment:
I'm not sure here. Wouldn't a markup return something like "&lt;p&gt;&lt;/p&gt;" for an empty string ?
Shouldn't that be an else statement instead ?